### DISCUSSION TITLE
Q: 为什么我一注册就报错500？ || Q: Why do I get an error 500 when I register?

### DISCUSSION THREAD

**Discussion by 0zjn0123** [walinejs/waline#2581] [Q&A]
### Describe the bug | 问题描述

注册账号就报500
![捕获](https://github.com/walinejs/waline/assets/116432703/5eca9163-0a7c-47bb-8843-1b8effbfd108)
我是在自己的服务器上搞的不是云部署
用的是nodejs部署

### Website URL | 问题网站

http://autozwaline.hp2.hpnu.cn

### Where your waline deploy? | 服务部署在哪里？

Docker

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

Register an account and claim 500
![Capture](https://github.com/walinejs/waline/assets/116432703/5eca9163-0a7c-47bb-8843-1b8effbfd108)
I did it on my own server, not cloud deployment.
Using nodejs deployment

### Website URL | Problem website

http://autozwaline.hp2.hpnu.cn

### Where your waline deploy? | Where is the service deployed?

Docker

### Where your comment data store? | Where is the data stored?

GitHub

**Comment by lizheming** [comment]
没有配置环境变量么，报了 LeanCloud 未初始化的错误

---
Are there no environment variables configured? LeanCloud is not initialized error reported
